index.html

```diff
@@ -1,21 +1,266 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="script.js" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="about.html" />
+    <link rel="stylesheet" href="contact.html" />
```

Comment:
Lines 7, 9 and 10 are not doing anything. We only use this format `<link rel="stylesheet" href="style.css" />` for our CSS files. You should delete them.